BrainPortal/app/models/cluster_task.rb

```diff
@@ -2383,10 +2383,15 @@ def singularity_commands(command_script)
     # Some of them might be patterns (e.g. /a/b/data*.squashfs) that need to
     # be resolved locally.
     # This will be a string "--overlay=path:ro --overlay=path:ro" etc.
-    overlay_paths = self.tool_config.singularity_overlays_full_paths.map do |path|
-      paths = Dir.glob(path) # checks on the local file system
-      cb_error "Can't find any overlays matching '#{path}'" if paths.blank?
-      paths
+    overlay_paths = self.tool_config.singularity_overlays_full_paths.map do |knd, id_or_name, paths|
```

Comment:
This is a bit too vervose. We don't need one line to say "processing" (line 2387), then one line that says "checking" (2389) just to finally get the line that we want, at 2492. Limit yourself to a single, simpler:

```
"Overlay configured: {type} {path}"
```

Reply:
For userfiles would you prefer log database id or path is ok?

Reply:
Just the basename of the file is fine.